specifyweb/frontend/js_src/lib/components/Toolbar/DataModel.tsx

```diff
@@ -170,7 +170,7 @@ function DataModelTable({
   ) : (
     <section className="flex flex-col gap-4" ref={forwardRef}>
       <div className="flex items-center gap-4">
-        <div className="flex items-center gap-2">
+        <div className="flex items-center gap-2" id={tableName}>
           <TableIcon label={false} name={model.name} />
           <H2 className="text-2xl" id={model.name.toLowerCase()}>
```

Comment:
Since you added ID to <div>, it should be removed from the header. (or maybe the ID should be removed from the div in favor of the header - like it was before)